### DISCUSSION TITLE
beam search and context graph in offline-whisper-decoder

### DISCUSSION THREAD

**Discussion by lucas-jo** [k2-fsa/sherpa-onnx#671] [Q&A] [answered]
Hello! Thank you for your great work!

I'd like to know if there is any plan or idea to implement beam search and context graph utilization from offline whisper decoder?

Thank you.

**Comment by csukuangfj** [comment, marked as answer]
Currently, we have only implemented greedy_search.

The results seem to be reasonable with greedy search and there is no plan to add other decoding methods for whisper, though we are open to community contributions to add other decoding methods for whisper.
